### ISSUE TITLE
Error in gem push workflow action

### ISSUE THREAD

**Issue by bobmaerten** [levups/fuel_surcharge#32]
```
rake aborted!
ERROR:  While executing gem ... (Gem::Package::FormatError)
    No such file or directory @ rb_sysopen - 

Tasks: TOP => release:rubygem_push
(See full trace by running task with --trace)
Bundled gems are installed into `/usr/local/bundle`
```

**Comment by bobmaerten** [comment]
Seem the build from the previous task is not persisted?

**Comment by bobmaerten** [comment]
tried running `rake build release:rubygem_push` in the same action but It fails too because `release:rubygem_push` is a "private" task :/

**Comment by bobmaerten** [comment]
closed by #33
